apps/cowswap-frontend/src/legacy/state/price/updater.ts

```diff
@@ -195,6 +319,12 @@ export default function FeesUpdater(): null {
     const fromDecimals = sellCurrency?.decimals ?? DEFAULT_DECIMALS
     const toDecimals = buyCurrency?.decimals ?? DEFAULT_DECIMALS
 
+    const unsupportedToken =
+      Boolean(isUnsupportedTokenGp(sellCurrencyId)) || Boolean(isUnsupportedTokenGp(buyCurrencyId))
```

Comment:
Nitpick: Are booth `Boolean` needed?

```suggestion
    const unsupportedToken =
      Boolean(isUnsupportedTokenGp(sellCurrencyId) || isUnsupportedTokenGp(buyCurrencyId))
```

Reply:
it has a weird type:
```
const isUnsupportedTokenGp: (address?: string | undefined) => false | {
    address: string;
    dateAdded: number;
}
```

If its a boolean, is Boolean(value) --->  `true` | `false` 
If its an object, it means is supported, but because we want a boolean, `Boolean({ address: ... })` ---> `true`

There's other ways of achiving the same, but this one was straigh fw